target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -48,16 +48,16 @@ endef
 TARGET_DEVICES += buffalo_wxr-5950ax12
 
 define Device/compex_wpq873
-       $(call Device/FitImage)
```

Comment:
Please all of the indent fixes, reordering etc into separate commits.

Reply:
Is it okay to add these commits to this PR?